eks/bastion.tf

```diff
@@ -1,18 +1,46 @@
 resource "aws_eip" "bastion_eip" {
   instance = aws_instance.bastion.id
-  vpc      = true
+  vpc = true
 }
 
 resource "aws_instance" "bastion" {
-  ami           = data.aws_ami.opszero_eks.id
+  //  ami = data.aws_ami.opszero_eks.id
+  ami = "ami-049aea444f70407b8"
   instance_type = "t2.micro"
 
-  key_name                    = var.ec2_keypair
+  key_name = var.ec2_keypair
   associate_public_ip_address = true
-  subnet_id                   = aws_subnet.public[0].id
-  vpc_security_group_ids      = [aws_security_group.node.id]
+  subnet_id = aws_subnet.public[0].id
+  vpc_security_group_ids = [
```

Comment:
This is a weird indent.

Reply:
I use intellij formatting, what do you use? and/or if you have style guides for tf <.<